dist/index.html

```diff
@@ -1,14 +1,199 @@
+</head>
+<body>
+<header>
+  <div class="header-container">
+    <div class="header-left">
+      <a href="#" title="Home">
+        <img src="images/header-logo-sml.png" alt="logo" class="header-small-logo-img">
+      </a>
+    </div>
+    <div class="header-right">
+      <div class="header-nav-wrapper">
+        <nav class="nav-wrap">
+          <div class="nav-item">
+            <a href="#">Become a host</a>
+          </div>
+          <div class="nav-item">
+            <a href="#">Trips</a>
+          </div>
+          <div class="nav-item">
+            <a href="#">Messages</a>
+          </div>
+        </nav>
+      </div>
+      <div class="header-profile-wrapper">
+        <a href="#" title="Profile" class="profile">
+          <img src="images/user-profile-sml.png" alt="profile" class="header-logo">
+        </a>
+      </div>
+    </div>
+  </div>
+</header>
+
+<main>
+  <a href="#" title="Home-foto">
+    <img src="images/main-image-big.png" alt="Home-foto" class="main_photo">
+  </a>
+  <div class="main-container">
+    <div class="main-info-block">
+      <div class="apartment-info">
+        <h1>Quiet apartment</h1>
+        <div class="apartment_type">Entire apartment</div>
+        <div class="apartment-info-parametres">
+          <div>4 guests</div>
+          <div>1 bed</div>
+          <div>1 bath</div>
+        </div>
+        <div class="apartment_location">
+          <div  class="descriptive_text">
+            City Apartment is newly renovated in the heart of Budapest,
+            right in the city center. Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit, sed do eiusmod tempor
+            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
+          </div>
+        </div>
+        <div class="contact-host"><a href="#" class="is-bold">Contact host</a></div>
+
+
+        <div class="apartment-amenities">
+          <h2 class="is-bold">Amenities</h2>
+
+          <div class="amenities_container">
+            <div class="amenities_item">Elevator</div>
+            <div class="amenities_item">Washer</div>
+            <div class="amenities_item">Hair dryer</div>
+            <div class="amenities_item">Kitchen</div>
+          </div>
+<!--
+          <div class="amenities-list-container">
+            <div class="amenities-list-container-col1">
+              <p>Elevator</p>
+              <p>Washer</p>
+            </div>
+            <div class="amenities-list-container-col2">
+              <p>Hair dryer</p>
+              <p>Kitchen</p>
+            </div>
+          </div>-->
+
+          <div class="show-all is-bold"><a href="#">Show All Amenities</a></div>
+        </div>
+      </div>
+
+      <div class="booking-info-block">
+        <div class="booking_price booking_input_wrapper">25$ per night</div>
+        <div class="booking_input_wrapper">
+          <input type="text" name="date" id="date" placeholder="Date" class="booking_input">
```

Comment:
should be a date input

Reply:
we have no idea how to style date input in way as it is on mockup